test/internals/reply.test.js

```diff
@@ -6,6 +6,7 @@ const sget = require('simple-get').concat
 const http = require('http')
 const NotFound = require('http-errors').NotFound
 const Reply = require('../../lib/reply')
+const responseSchema = require('../../lib/validation').responseSchema
```

Comment:
This should actually be `require('../../lib/validation').schema.responseSchema`. However, I'm pretty sure that if you do that the test will start to fail (because the serializer expects the responseSchema to be a function, not an object).

Here's what I'm going to suggest:
Remove the `responseSchema` stuff and use an empty object for the context: `const context = {}`. That will test the serializer using the fallback `JSON.stringify()`. Then make another test that creates a `fastify` instance and a route with a response schema that uses `reply.serialize()`, then use `fastify.inject()` to test the route ([here's a sort of similar example](https://github.com/fastify/fastify/blob/master/test/shared-schemas.test.js#L77-L94)).